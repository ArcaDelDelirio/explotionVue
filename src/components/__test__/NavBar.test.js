import { shallowMount, createLocalVue } from '@vue/test-utils';
import VueRouter from 'vue-router';
import NavBar from '../NavBar';

describe('NavBar View', () => {
  it('Render the component', () => {
    const localVue = createLocalVue();
    localVue.use(VueRouter);
    const router = new VueRouter();
    const wrapper = shallowMount(NavBar, {
      localVue,
      router,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });
});
