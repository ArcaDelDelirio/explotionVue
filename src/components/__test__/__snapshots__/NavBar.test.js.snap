// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar View Render the component 1`] = `
<header>
  <nav class="navbar">
    <div class="logo-container"><span class="logo"><router-link-stub to="/" tag="a" event="click"><img src="@/assets/movie.png" height="24px" width="24px" alt="logo"></router-link-stub></span> <span>Super Movie DataBase</span></div>
    <div class="login-container">
      <div class="login-state">
        <div class="login-caps">
          <router-link-stub to="/login" tag="button" event="click" class="sign-button">Sign In</router-link-stub>
        </div>
        <!---->
      </div>
    </div>
  </nav>
</header>
`;
